packages/wonder-blocks-tabs/src/components/tab-panel.tsx

```diff
@@ -48,12 +49,16 @@ export const TabPanel = (props: Props) => {
     } = props;
 
     const ref = React.useRef<HTMLDivElement>(null);
-    const [hasFocusableElement, setHasFocusableElement] = React.useState(false);
+    // Initialize to null to indicate that the focusable element status is not
+    // yet determined
+    const [hasFocusableElement, setHasFocusableElement] = React.useState<
+        boolean | null
+    >(null);
 
     React.useEffect(() => {
         // Whenever tab panel contents change, determine if the panel has a
-        // focusable element
-        if (ref.current) {
+        // focusable element (only if the tab panel has children)
+        if (ref.current && children) {
```

Comment:
If `children` is falsey, it means that the tab panel content hasn't been rendered yet since the Tabs component renders the panel content conditionally 

https://github.com/Khan/wonder-blocks/blob/0bb964892a681a14471cc289501a9e8fd8125812/packages/wonder-blocks-tabs/src/components/tabs.tsx#L395-L412

This additional check makes it so the tab index on the tabpanel is not initialized until the `children` prop is provided with content. This prevents the FocusManager in Popover from including inactive tabpanels in it's list of focusable elements 

(related to https://github.com/Khan/wonder-blocks/pull/2863/files#r2528858860)